corgidrp/mocks.py

```diff
@@ -1,12 +1,204 @@
 import astropy.io.fits as fits
 from astropy.time import Time
 import numpy as np
+import os
 
 import corgidrp.data as data
 import corgidrp.detector as detector
-import os
+from corgidrp.detector import imaging_area_geom, unpack_geom
+
+detector_areas_test= {
+'SCI' : { #used for unit tests; enables smaller memory usage with frames of scaled-down comparable geometry
+        'frame_rows' : 120,
+        'frame_cols' : 220,
+        'image' : {
+            'rows': 104,
+            'cols': 105,
+            'r0c0': [2, 108]
+            },
+        'prescan' : {
+            'rows': 120,
+            'cols': 108,
+            'r0c0': [0, 0]
+            },
+        'prescan_reliable' : {
+            'rows': 120,
+            'cols': 108,
+            'r0c0': [0, 0]
+            },
+        'parallel_overscan' : {
+            'rows': 14,
+            'cols': 107,
+            'r0c0': [106, 108]
+            },
+        'serial_overscan' : {
+            'rows': 120,
+            'cols': 5,
+            'r0c0': [0, 215]
+            },
+        },
+'ENG' : { #used for unit tests; enables smaller memory usage with frames of scaled-down comparable geometry
+        'frame_rows' : 220,
+        'frame_cols' : 220,
+        'image' : {
+            'rows': 102,
+            'cols': 102,
+            'r0c0': [13, 108]
+            },
+        'prescan' : {
+            'rows': 120,
+            'cols': 108,
+            'r0c0': [0, 0]
+            },
+        'prescan_reliable' : {
+            'rows': 220,
+            'cols': 20,
+            'r0c0': [0, 80]
+            },
+        'parallel_overscan' : {
+            'rows': 116,
+            'cols': 105,
+            'r0c0': [104, 108]
+            },
+        'serial_overscan' : {
+            'rows': 220,
+            'cols': 5,
+            'r0c0': [0, 215]
+            },
+        }
+}
+
```

Comment:
Same as above, for all the testing more descriptive variable names would be ideal